src/common/snippets/src/pass/insert_movebroadcast.cpp

```diff
@@ -65,7 +65,7 @@ std::shared_ptr<ngraph::Node> numpy_broadcast_node(const ngraph::Output<ngraph::
 
     if (do_broadcast) {
         // ShapeOf
-        broadcasted_node = std::make_shared<ngraph::snippets::op::BroadcastMove>(broadcasted_node, output_shape);
+        return std::make_shared<ngraph::snippets::op::BroadcastMove>(value, output_shape);
```

Comment:
What exactly the problem is solved here? Just wondering is it possible to cover the case by test?

Reply:
1) `broadcasted_node` returns default port, but if `broadcasted_node` has two output ports then it will fail
2) Right now snippets don't support operations with two and more outputs. Can be done after `Split` (for example) implementation.